controllers/operatorconfig_controller.go

```diff
@@ -34,15 +33,17 @@ type OperatorConfigReconciler struct {
 	Scheme *runtime.Scheme
 }
 
-// rbac.authorization.k8s.io permissions are needed to create namespace limited role and rolebinding to create deployment and service within mtk-operator
+// rbac.authorization.k8s.io permissions are needed to create namespace limited role and rolebinding to create deployment and service within openshift-migration
 //+kubebuilder:rbac:groups=crane.konveyor.io,resources=operatorconfigs,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=crane.konveyor.io,resources=operatorconfigs/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=crane.konveyor.io,resources=operatorconfigs/finalizers,verbs=update
 //+kubebuilder:rbac:groups=tekton.dev,resources=clustertasks,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=roles;rolebindings,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups="apps",namespace=mtk-operator,resources=deployments,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups=console.openshift.io,resourceNames=crane-ui-plugin,resources=consoleplugins,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups="",namespace=mtk-operator,resources=services,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=roles;rolebindings;clusterroles;clusterrolebindings,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups="apps",namespace=openshift-migration,resources=deployments,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=console.openshift.io,namespace=openshift-migration,resources=consoleplugins,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups="",namespace=openshift-migration,resources=services;serviceaccounts,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=route.openshift.io,namespace=openshift-migration,resources=routes,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups="",namespace=openshift-migration,resources=secrets,verbs=get;list;watch;create;update;patch;delete
```

Comment:
We can probably drop some of these now right?

At a minimum we should be able to drop secrets.